### PR TITLE
4533 new tab

### PR DESCRIPTION
**Pull request by mrinsin**
image

awaiting further UI details from Jessica & Kristen

### CODE REVIEW

File: web-client/src/presenter/computeds/caseDetailHeaderHelper.js [hunk 1]
```diff
@@ -52,7 +52,11 @@ export const caseDetailHeaderHelper = (get, applicationContext) => {
 
   const showCreateOrderButton = permissions.COURT_ISSUED_DOCUMENT;
 
-  const showAddDocketEntryButton = permissions.DOCKET_ENTRY;
+  const hasDocketEntryPermission = permissions.DOCKET_ENTRY;
+
+  const showAddDocketEntryButton = hasDocketEntryPermission;
+
+  const showNewTabLink = hasDocketEntryPermission;
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/caseDetailHeaderHelper.js]: Should this be for all internal users?

File: web-client/src/views/CaseDetail/CaseDetailHeader.jsx [hunk 2]
```diff
@@ -109,7 +109,8 @@ export const CaseDetailHeader = connect(
                       {caseDetailHeaderHelper.showFileDocumentButton && (
                         <Button
                           secondary
-                          className="tablet-full-width push-right margin-right-0"
+                          className="margin-right-05"
+                          // className="tablet-full-width push-right margin-right-0"
```
Comment (rachaelparris) [inline web-client/src/views/CaseDetail/CaseDetailHeader.jsx]: Can remove? Should this button need any changes? I think this only displays for external users, and they wouldn't see the New Tab button.

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/CaseDetail/CaseDetailHeader.jsx (hunk 2 above)
Comment (mrinsin) [reply web-client/src/views/CaseDetail/CaseDetailHeader.jsx]: oops sorry I think this was me forgetting to clean up, I think it needs to be reverted!

File: web-client/src/presenter/computeds/caseDetailHeaderHelper.js (hunk 1 above)
Comment (mrinsin) [reply web-client/src/presenter/computeds/caseDetailHeaderHelper.js]: Thank you!!!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**